### ISSUE TITLE
[Bug] - Negative Age in the Profile Section. 

### ISSUE THREAD

**Issue by rsc1102** [gcivil-nyu-org/wed-fall24-team3#130]
**Description**
In the profile section, the age can be set to a negative number. 

**Steps to Reproduce**
1. Log in
2. Click on "Profile".
3. Set the age to a negative number.
4. Click "Update Profile".

**Expected Behavior**
User should not be able to set a negative age in their profile. 

**Screenshots or Code Snippets (required)**
![image](https://github.com/user-attachments/assets/17c074cd-0fef-4d1c-b2d5-eb2eb6edd9e4)

**Comment by simarmehta** [comment]
Good catch @rsc1102 , we need to set it up such that only positive integers are allowed, will be fixed before next sprint! Thanks!
